cli.js

```diff
@@ -1,33 +1,50 @@
 #!/usr/bin/env node
 
-const { Command, Argument } = require('commander');
+const { Command } = require("commander");
 const program = new Command();
-const { configPath } = require('./utils/getConfigPath');
-const { translateController } = require('./controllers/translateController');
-const { languagesController } = require('./controllers/languagesController');
-const { generateConfigController } = require('./controllers/generateConfigController');
+const { configPath } = require("./utils/getConfigPath");
+const { translateController } = require("./controllers/translateController");
+const { languagesController } = require("./controllers/languagesController");
+const {
+  generateConfigController,
+} = require("./controllers/generateConfigController");
 
 program
-  .name('i18n-populator')
-  .description('CLI to translate JavaScript strings')
-  .version('1.0.0');
+  .name("i18n-populator")
+  .description("CLI to translate JavaScript strings")
+  .version("1.0.0");
```

Comment:
What about using the version from package.json?

Reply:
done. Thanks for the suggestion!